packages/web3-test/src/index.ts

```diff
@@ -141,6 +143,13 @@ export async function transfer(from: PrivateKeyWallet, to: Address, tokenId: str
   return await from.signAndSubmitTransferTx({ signerAddress: from.address, destinations: [destination] })
 }
 
+export async function mintToken(from: PrivateKeyWallet, amount: bigint) {
+  const nodeProvider = tryGetDevnetNodeProvider()
+  const result = await createAndTransferToken(nodeProvider, from, amount)
+  const contractId = await getContractIdFromUnsignedTx(nodeProvider, result.unsignedTx)
+  return { ...result, contractId, contractAddress: addressFromContractId(contractId) }
+}
```

Comment:
let's create `const.ts` and then move this function to `token.ts`. We don't need to expose `createAndTransferToken` I think.

Reply:
I added an index.ts file for exporting what we want to export.